components/http_foundation.rst

```diff
@@ -712,6 +712,34 @@ Session
 
 The session information is in its own document: :doc:`/components/http_foundation/sessions`.
 
+UrlHelper
+-------
+
+Generating absolute (and relative) URLs for a given path is a common need in lots of applications. In Twig templates this is trivial thanks to the absolute_url() and relative_path() functions. The same functionality can be found in the :class:`Symfony\\Component\\HttpFoundation\\UrlHelper` class, which can be injected as a service anywhere in your application. This class provides two public methods called getAbsoluteUrl() and getRelativePath()::
```

Comment:
on sf doc, it is tried to keep lines to ~ 70/80 chars then go to new line :)

Reply:
2c6bb4e